modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee

```diff
@@ -911,27 +911,46 @@ class FileRowSummaryController extends Backbone.View
 	className: 'dataTableRow'
 	events:
 		"click": "handleClick"
-
+	
 	handleClick: =>
 		@trigger "gotClick", @model
 		$(@el).closest("table").find("tr").removeClass "info"
 		$(@el).addClass "info"
 
 	initialize: ->
 		@template = _.template($('#FileRowSummaryView').html())
-
+	
 	render: =>
 		fileDate = @model.get('fileDate')
 		if fileDate is null
 			fileDate = ""
 		else
 			fileDate = UtilityFunctions::convertMSToYMDDate(fileDate)
+
+		#To get the current state, we need the bulk file ID to get lots and then generate the updated SDF file
+		reportID = @model.get('id')
+		fileName = @model.get('fileName')
+
+		# Rename the SDF file representing the current state
+		currentFileName = fileName.replace "\.sdf", "_current_state.sdf"
```

Comment:
Thinking out loud - I wonder if we should add a date or timestamp to the filename. It could be useful if someone's downloaded this several times to know at what point this was the current state.